docs/contributing.md

```diff
@@ -32,29 +32,61 @@ You can check that everything compiles by running the following from the
 root of your checkout:
 
 ```
-cargo test --all
+make test-rust
 ```
 
 If you plan to work on the Android component bindings, you should also review
 the instructions for [setting up an Android build environment](https://github.com/mozilla/glean/blob/master/docs/dev/setup-android-build-environment.md)
 
+To run all Kotlin tests:
+
+```
+make test-kotlin
+```
```

Comment:
"or run tests in Android Studio."  (I'm usually a CLI person as well, but the Android Studio test runner is really convenient for debugging a failing test...)